flink-runtime/pom.xml

```diff
@@ -117,6 +117,16 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
```

Comment:
For the sake of completeness, here is also the corresponding IntelliJ issue: https://youtrack.jetbrains.com/issue/IDEA-107048